components/dashboard/src/projects/NewProject.tsx

```diff
@@ -0,0 +1,314 @@
+                        Select a Git provider first and continue with your repositories.
+                    </div>
+                    <div className="mt-6 flex flex-col space-y-3 items-center">
+                        {authProviders.map(ap => {
+                            return (
+                                <button key={"button" + ap.host} className="btn-login flex-none w-56 h-10 p-0 inline-flex" onClick={() => selectProvider(ap)}>
+                                    {iconForAuthProvider(ap.authProviderType)}
+                                    <span className="pt-2 pb-2 mr-3 text-sm my-auto font-medium truncate overflow-ellipsis">Continue with {simplifyProviderName(ap.host)}</span>
+                                </button>
+                            );
+                        })}
+                    </div>
+                </div>
+            </>)}
+            {!showGitProviders && (<>
+                <div className="p-6">
+                    <div className="flex flex-col space-y-2">
+                        <label htmlFor="selectAccount" className="font-medium">Account</label>
+                        <select name="selectAccount" value={selectedAccount} className="w-full"
+                            onChange={(e) => onSelectAccount(e.target.value)}>
+                            {accounts.map(a => (<option key={`account-${a}`} value={a}>{`${a} (${provider})`}</option>))}
+                            {isGitHub() && (<option value="addOrg">Add GH Org</option>)}
+                            <option value="selectProvider">Select Git Provider</option>
+                        </select>
+                    </div>
+                    <div className="flex-col mt-4">
+                        {reposToRender.length > 0 && (
+                            <div className="overscroll-contain max-h-96 overflow-y-auto">
+                                {reposToRender.map(r => (
+                                    <div key={`repo-${r.name}`} className="rounded-md whitespace-nowrap flex space-x-2 py-2 px-3 w-full justify-between hover:bg-gray-100 dark:hover:bg-gray-800 focus:bg-gitpod-kumquat-light transition ease-in-out group">
+                                        <div className="w-8/12 text-gray-400 flex flex-col">
+                                            <span className="text-gray-800 dark:text-gray-300 text-base font-semibold overflow-ellipsis truncate">{toSimpleName(r.name)}</span>
+                                            <span className="">Updated __ ago</span>
+                                        </div>
+                                        <div className="w-4/12 flex justify-end">
+                                            <div className="flex self-center hover:bg-gray-200 dark:hover:bg-gray-700 rounded-md cursor-pointer opacity-0 group-hover:opacity-100">
+                                                <button className="primary py-1" onClick={() => selectRepo(r)}>Select</button>
```

Comment:
would that apply to the `Select Team` page as well?

Reply:
In case the project is already part of a team we could trigger a modal with a CTA button to request access to the team, instead of having conditional buttons here _Add Project_ and _Select Team_. WDYT?